<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addInfoWindow</class>
 <widget class="QMainWindow" name="addInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="mention_addInfo">
      <property name="text">
       <string>Confirm you have saved before back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_name" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="NAME_ADDINFOWINDOW">
         <property name="text">
          <string>NAME:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_number" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="NUMBER_ADDINFOWINDOW">
         <property name="text">
          <string>NUMBER:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_number_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_position" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="POSITION_ADDINFOWINDOW">
         <property name="text">
          <string>POSITION:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_position_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_pay" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="PAY_ADDINFOWINDOW">
         <property name="text">
          <string>PAY:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pay_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_worktime_OnlyTEC" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="WORKTIME_ADDINFOWINDOW">
         <property name="text">
          <string>WORKTIME:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_worktime_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_sales_OnlySALESMAN" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="SALES_ADDINFOWINDOW">
         <property name="text">
          <string>SALES:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_sales_addInfoWindow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_managedPeople_OnlySALESMANAGER" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="MANAGEDPEOPLE_ADDINFOWINDOW">
         <property name="text">
          <string>MANAGED PEOPLE(Number)ï¼š</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_addManagedPeople_addInfo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addManagedPeople_button_addInfoWindow">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finfish_addManaged_addInfo">
         <property name="text">
          <string>Finish</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionBack_addInfoWindow"/>
    <addaction name="actionSave_addInfoWindow"/>
   </widget>
   <addaction name="menuSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBack_addInfoWindow">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionSave_addInfoWindow">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveInNewFile_addInfoWindow">
   <property name="text">
    <string>Save in NewFile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
